search-provider/eks-discovery-feed-provider.c

```diff
@@ -530,7 +530,7 @@ article_card_descriptions_cb (GObject *source,
 
           while (g_variant_iter_loop (&discovery_feed_content_iter, "{sv}", &key, &value))
             {
-              if (g_strcmp0 (key, "blurbs") == 0)
+              if (g_strcmp0 (key, "hookTitle") == 0)
```

Comment:
See my comment in https://github.com/endlessm/eos-subscriptions-apps/pull/309 -- I would not rename this field from `blurbs`